apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -8527,6 +8525,10 @@ button.btn.btn-filter {
     margin-left: 10px !important;
 }
 
+.td_in_searchBox {
```

Comment:
Avoid using snake case in class names

Reply:
skipping for now, this class appear in many pages